googlecloud/cloudstorage-client/src/test/java/org/webpieces/googlecloud/storage/TestLocalStorage.java

```diff
@@ -113,13 +134,49 @@ public void addFileToBucketAndThenListFiles() {
     }
 
     @Test
-    public void addFileThenReadThenDeleteThenListFiles() {
+    public void addFileThenReadThenDeleteThenListFiles() throws IOException {
+        writeFile(BlobId.of("testbucket", "mytest1.txt"));
+        GCPBlob bucket = instance.get("testbucket", "mytest1.txt");
+        Assert.assertEquals("mytest1.txt",bucket.getName());//passed.
+
+        ReadableByteChannel readFile = instance.reader("testbucket", "mytest1.txt");
+
+        InputStream i = Channels.newInputStream(readFile);
 
+        String text = new BufferedReader(
+                new InputStreamReader(i, StandardCharsets.UTF_8))
+                .lines()
+                .collect(Collectors.joining("\n"));
+        Assert.assertEquals("testing a bitch", text);// Passed.
+
+        boolean success = instance.delete("testbucket", "mytest1.txt");
+        Assert.assertEquals(false,success);//passed.
```

Comment:
This should be 

Assert.assertEquals(true,success); or simply Assert.assertTrue(success) as the file should be deleted and false implies it was not deleted.